pkg/deploy/lattice/target_group_manager_test.go

```diff
@@ -77,13 +80,12 @@ func Test_CreateTargetGroup_TGNotExist_Active(t *testing.T) {
 			Status: &tgStatus,
 		}
 		p := int64(8080)
-		prot := "HTTP"
 		emptystring := ""
 		config := &vpclattice.TargetGroupConfig{
 			Port:            &p,
-			Protocol:        &prot,
+			Protocol:        &tgSpec.Config.Protocol,
 			VpcIdentifier:   &config.VpcID,
-			ProtocolVersion: &emptystring,
+			ProtocolVersion: &tgSpec.Config.ProtocolVersion,
```

Comment:
Are Protocol and ProtocolVersion required now when they weren't before or are empty values just an artifact of the unit tests?